gulpfile.js

```diff
@@ -254,23 +256,48 @@ gulp.task('generate-plunks', function() {
   return gulpFile(plunks, {src: true}).pipe(gulp.dest('demo/src/public/app/components'));
 });
 
-gulp.task('clean:demo', function() { return del('demo/dist'); });
+gulp.task('clean:demo', function() {
+
+  spawn('npm', ['uninstall', '@ng-bootstrap/ng-bootstrap'], {stdio: 'inherit'}).on('error', function(err) {
+    console.log('Failed to run npm uninstall @ng-bootstrap/ng-bootstrap : ' + err);
```

Comment:
Change to `console.error`, remove space before `:`, and change to template literal, i.e.
```js
`Failed to run npm uninstall @ng-bootstrap/ng-bootstrap: ${err}`
```